### ISSUE TITLE
In the YoloV1 boxes competitive with each other

### ISSUE THREAD

**Issue by ermolenkodev** [learnml-today/object-detection-with-pytorch#12]
What do you mean by the sentence 'In the YoloV1 boxes competitive with each other'?? Is it because of the loss function?

**Comment by creotiv** [comment]
there are 2 bboxes in YoloV2 (by default) and they are not set to some specific scale like in SSD or next version of Yolo where using Anchor Boxes. So basically you have 2 bboxes that trying to split distribution of different positions as best as they have. So they can compete for one result.

**Comment by ermolenkodev** [comment]
I got it. Thanks
